libvmi/os/windows/core.c

```diff
@@ -50,6 +51,8 @@ win_ver_t ntbuild2version(uint16_t ntbuildnumber)
         case 9200:
         case 9600:
             return VMI_OS_WINDOWS_8;
+        case 18432:
```

Comment:
I'm getting weird values occasionally, sometimes under 9000. Not sure what's up with that so I just turned this check into a warning.